File: csharp/etl/Etl.cs
using System.Collections.Generic;

public class Etl
{
    public static Dictionary<string, int> Transform(Dictionary<int, IList<string>> old)
    {
        var output = new Dictionary<string, int>();

        foreach(var score in old.Keys)
        {
            foreach (var letter in old[score])
            {
                output.Add(letter.ToLower(), score);
            }
        }

        return output;
    }
}

File: swift/etl/ETL.swift
class ETL {
    static func transform(_ old: [Int: [String]]) -> [String:Int] {
        var output : [String : Int] = [:]
        for (points, chars) in old {
            for char in chars {
                output[char.lowercased()] = points
            }
        }
        return output
    }
}
